### DISCUSSION TITLE
'path' command cannot search for files that do not display content, such as 'zip' or 'png'

### DISCUSSION THREAD

**Discussion by XDYB** [community/community#55367] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body
2023-05-14
The latest search syntax discards 'extension' and 'filename' and replaces it with 'path', but the 'path' syntax cannot search for files that do not directly display content, such as '.zip' or '.png', 'path' Only files with displayed content can be searched, such as ".txt" or ".md".

filename:.zip≠path:\*.zip
extension:.zip≠path:\*.zip

Can't search for real files.

what should I do? 
i need your thoughts.
Thanks.

2023-05-16
After my exploration, is it because the current search only supports UTF8? Can search in other formats be supported as soon as possible, or the provider provides the function of switching to the old version

**Comment by look** [comment]
Non-UTF-8 files are not in the index at all, so you can't search for them by path. Sorry for breaking your workflow, we hope to bring this back, but I can't promise when we'll be able to do that.
